index.html

```diff
@@ -6,24 +6,35 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-    <title>LinuxChix India</title>
-    <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Arima+Madurai">
+    <title>LinuxChix India</title>
+
+    <!-- SEO Tags -->
+    <meta name="description" content="Official Website of LinuxChix India">
+    <meta name="keywords" content="LinuxChix, LinuxChix India, LinuxChixIn, linuxchix, linuxchixin, lci, LCI">
+
+    <!-- Main CSS File -->
+    <link rel="stylesheet" href="assets/css/main.css">
```

Comment:
`main.css` file should override the rest of the theme. Can you please fix it as it was. Categorizing bunch of imports is nice but do not change the order.